config/software/libxml2.rb

```diff
@@ -33,6 +33,10 @@
   source md5: "9c0cfef285d5c4a5c80d00904ddab380"
 end
 
+version "2.7.8" do
```

Comment:
the old versions should probably be cleaned out.

after all, they've got CVEs against them.

if that breaks something, then it would imply there's an override pinning the version, which would be a bad thing to happen since we'd be pinning on old, vulnerable versions.

if dropping all the old versions breaks things in a way that can't be solved by removing a version pin, then the old versions can get re-introduced and versions can then get pinned after we determine there's a real $REASON for it.